src/typing/statement.ml

```diff
@@ -4078,6 +4041,55 @@ and subscript =
           lhs, lhs_t, chain, ast
         end
 
+    | Member {
```

Comment:
This block was just relocated to prevent it from prematurely capturing the case you're looking at, right? Just checkin

Reply:
Not sure about it, I added one more case before, but I removed it because it seemed to do nothing https://github.com/facebook/flow/commit/5650cc2a355ddafc3385d9ec856c452f60a19528